### ISSUE TITLE
The ARIA attribute "aria-label" is not valid on <div> with implicit ARIA role "generic"

### ISSUE THREAD

**Issue by fkopp-hipsquare** [jonnitto/Jonnitto.PrettyEmbedHelper#34]
### Describe the bug

The wrapping elements of the audio and video players are `<div>`s with `aria-label` attributes but no appropriate `role` attribute. That is forbidden according to [this MDN documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/generic_role).

This poses an issue for screen readers and other assistive technology, which might be unable to work with the players as intended. In turn it limits the use of PrettyEmbed in projects which want or require WCAG comformity.

### Expected behaviour

The wrapping elements of the audio and video players should be `<aside>`s instead of `<div>`s. Basically they would work the same as `<div>`s, but their role would be "complementary" instead of "generic", allowing the `aria-label` attribute and also giving an appropriate hint to the role of the element in the page (see [this MDN blog about landmark roles](https://developer.mozilla.org/en-US/blog/aria-accessibility-html-landmark-roles/#the_complementary_landmark))

Alternative, but less ideal, solutions would be to add `role="complementary"` attributes to the existing `<div>`s, or to remove the `aria-label` attributes

### Steps to reproduce

1. Scan any page with a PrettyEmbed video or audio player with the [IBM Equal Access Accessibility Checker](https://chromewebstore.google.com/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp?hl=en-US)
2. Find the accessibility violation 'The ARIA attributes "aria-label" are not valid for the element <div> with implicit ARIA role "generic"' in the scan output on the wrapping `<div>` of the player.
3. In the developer tools, replace the div of the player with an aside
4. Scan again and note that the accessibility violation disappears with no change in player behavior and appearance

### Version

6.7.0

### PHP version

```shell
PHP 8.2.28 (cli) (built: Apr  8 2025 01:23:03) (NTS)
Copyright (c) The PHP Group
```

### Environment

```shell
omitted
```

### Additional context

If at all possibile I would be very grateful if this issue could also be fixed in v6

**Comment by jonnitto** [comment]
@fkopp-hipsquare Can you check my commit I made?

**Comment by fkopp-hipsquare** [comment]
@jonnitto Perfect, thank you so very much! 💖
